### PR TITLE
Add commuter rail colors and cleanup

### PR DESCRIPTION
**Pull request by devinmatte**
## Motivation

Prepare to add commuter rail to the dashboard in the future, by adding the colors and the sidebar selector ready to go

## Changes

- Adding commuter rail keys and colors
- Adding a commuter rail tab to codebase, but not adding it to the UI yet
- Fixes Bus sidebar icon

## Testing Instructions

There's nothing currently to test. Nothing is set to display

### CODE REVIEW

File: common/components/nav/CommuterRailDropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { SidebarTabs } from '../../../modules/navigation/SidebarTabs';
+import { TRIP_PAGES, COMMUTER_RAIL_OVERVIEW } from '../../constants/pages';
+import { CommuterRailRouteSelection } from './CommuterRailRouteSelection';
+
+interface CommuterRailDropdownProps {
+  close?: () => void;
+}
+
+export const CommuterRailDropdown: React.FC<CommuterRailDropdownProps> = ({ close }) => {
```
Comment (skaplan-dev) [inline common/components/nav/CommuterRailDropdown.tsx:10]: my take is we shouldn't use React.FC unless we're using typed children / the other props we get from it. It muddles the type for the component.

**Review by skaplan-dev — APPROVED** [review]
one nit

File: common/components/nav/CommuterRailDropdown.tsx (hunk 1 above)
Comment (devinmatte) [reply common/components/nav/CommuterRailDropdown.tsx:10]: I tend to disagree. I think it helps well label which functions are components vs just functions personally feel it reads cleaner

Comment (skaplan-dev) [reply common/components/nav/CommuterRailDropdown.tsx:10]: Everything is just functions 😛  its react baby

**Pull request merged**